CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file based on the
 
 * The `protocol` allowed value under `event.type` should not have the `expected_event_types` defined. #964
 * Clarify the definition of `file.extension` (no dots). #1016
+* Include `process.thread.name` in the experimental definitions. #1053
```

Comment:
I just now noticed that the changelog for the creation of the experimental artifacts was in the tooling section. However it's a big change schema-wise. I think we should move this here.

I'm saying this on this unrelated PR, because I'm not sure I would add a bugfix to something that hasn't been released yet. I was going to suggest to just append this PR number to the experimental artifacts changelog.

Something like the following, in the "Added" section for the schema:

```
* Added a new directory with experimental artifacts, which includes all changes
  from RFCs that have reached stage 2. #993, #1053
```

Reply:
++ I think this is a better change.